Public/Sdk/Public/Transformers/Transformer.Execute.dsc

```diff
@@ -162,6 +162,12 @@ namespace Transformer {
          */
         disableCacheLookup?: boolean;
 
+        /** 
+          * True if the executable depends on directories that comprise the current host OS. 
+          * For instance on windows this signals that accesses to the Windows Directories %WINDIR% should be allowed. 
+          */

```

Comment:
maybe add some comment about how this relates to `ToolDefinition.dependsOnWindowsDirectories`

Reply:
My hope is to have the same types on tool as execute. So that things remain flexibile for the user.

---
In reply to: [274228266](https://github.com/Microsoft/BuildXL/pull/125#discussion_r274228266) [](ancestors = 274228266)